src/pages/Books.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import BookCard from '../components/BookCard';
+
+export default function Books() {
+  return (
+    <>
+      <BookCard />
+      <BookCard />
+      <BookCard />
+      <form className="add-book-form">
+        <h2>ADD NEW BOOK</h2>
+        <input placeholder="Book title" className="input-title" required />
+        <select name="category" className="input-category" required>
+          <option defaultValue>Category</option>
+        </select>
+        <button type="submit" className="add-book">ADD BOOK</button>
+      </form>
+    </>
+  );
```

Comment:
To match the project requirements correctly, I'd like you to add and refactor some of the components:
- - [ ] Instead of rendering multiple `BookCard />` in the same page please move them to a new component (BookLisr).
- - [ ] Please move the form responsible or adding a new book to a separate component.

Reply:
Hi @omar-labana , 
Regarding the first point, I did separate the book into an independent component because I'm keeping in mind that I'll have a list of books from the API that I'll be interesting through to render each book. (The point of adding them multiple times was just to demonstrate the UI and how it'll look following the guideline).

Reply:
Coming to the second suggestion, I'll surely make the changes. Thanks a lot for that awesome point out @omar-labana  😊

Reply:
@blessedjasonmwanza I think there is a good reason for this, please take a look at react performance trick such as memoization.

Reply:
> @blessedjasonmwanza I think there is a good reason for this, please take a look at react performance trick such as memoization.

Oh - Thanks 😃 ! I will definitely do 👨🏽‍💻 🌐 ASAP.

> HappyCoding